### PR TITLE
Portfolio: validate contact form

### PR DESCRIPTION
**Pull request by odaialazzeh**
- [ ] Update the HTML file and add a label to show the error message.
- [ ] Update the CSS file with styles for error message.
- [ ] Update the javascript file with the code to validate the email.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1003,4 +1012,16 @@ nav.menu-btn ul li {
...
+    position: absolute;
+    color: #d8000c;
+    top: 10.5em;
+    right: 7em;
+    border: none;
+    padding-left: 1.5em;
+    font-size: 1.5em;
+    width: 10em;
+    border-radius: 0.5em;
+  }
```
Comment (mckent05) [inline style.css]: - You have not properly aligned your error message and it is causing the message to display incorrectly across various screen sizes, Please fix this 👍 👍 . I'd suggest you put your error message and button in a `div` tag and align with the `flexbox` property.

Your project:
error form 2
error form

File: function.js [hunk 2]
```diff
@@ -288,3 +288,16 @@ document.body.appendChild(div1);
...
+  const mailformat = /^[a-z]+@[a-z0-9-]+\.[a-z0-9-.]+$/;
+  if (inputText.value.match(mailformat)) {
+    return true;
+  }
+
+  const errorM = document.getElementById('error');
+  errorM.innerHTML = 'The email format is invalid';
+  document.form1.text1.focus();
+  return false;
+}
```
Comment (mckent05) [inline function.js:303]: - I am currently getting an error in the console(please check the picture attached below 👇 ), when I try to submit an email with all lowercase letters and also when I try to submit an email with one uppercase letter. Your function only runs and displays an error message when all letters are in uppercase. According to the project requirements, the email should only be submitted when all letters are in lowercase letters. Please try to fix this issue 👍 👍 

Your project(Please pay attention to the error in the console):
form submit valid

File: index.html [hunk 3]
```diff
@@ -129,17 +129,17 @@ <h3>Languages</h3>
...
                             <div class="input-name">
                                 <input type="text" name="First Name" id="First-Name" placeholder="First Name" minlength="1" maxlength="30" required>
                                 <input type="text" name="Last Name" id="Last-Name" placeholder="Last Name" minlength="1" maxlength="30" required>
                             </div>
-                            <input type="email" name="Email address" id="email" placeholder="Email address" required  pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$">
+                            <input type="email" name="text1" id="email" placeholder="Email address" required>
                             <textarea name="textarea" id="text-area" cols="30" rows="10" placeholder="Enter text here" required></textarea>
-                            <button type="submit">Get in touch</button>
-                            
+                            <button type="submit" onclick="ValidateEmail(document.form1.text1)">Get in touch</button>
```
Comment (mckent05) [inline index.html:141]: - For this project, according to the requirements, the `onClick` event listener should be implemented in the JS file, not in your HTML. Please correct this 👍 👍

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @girma336,

# STATUS: CHANGES REQUIRED ♻️ ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- Descriptive PR title and summary 👏 
- Your repo looks professional 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @odaialazzeh 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

Descriptive PR title and summary 👏
Your repo looks professional 👍 👍

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**